reservationWeekday.js

```diff
@@ -0,0 +1,34 @@
+const http = require('http');
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const hallsFile = require("./halls.js");
+var hallsAr = hallsFile.hallsArray;
+var { DateTime } = require('luxon');
+var _ = require('lodash');
+
+//this didint work...
+var newArray = _.filter(hallsAr, { 'reservation.isReserved' : true   });
```

Comment:
ako ne radi onda je kod dobro i obrisati ili zakomentirati

Reply:
var newArray = _.filter(hallsAr, { 'reservation' : { 'isReserved' : true }   });

Ovo je ispravan način, treba ulaziti objekt po objekt u ovom slučaju

Reply:
Hvala za komentare. Vidi se da studiozno proučavaš kod. :)

Reply:
Ovaj prvi mi je lapsus. A ovo drugo mi je sad jasno kako se to radi.